cmd/cspc-operator/app/start.go

```diff
@@ -35,16 +33,18 @@ import (
 )
 
 var (
-	kubeconfig string
+	kubeconfig = flag.String("kubeconfig", "", "Path for kube config")
 )
 
 // Start starts the cstor-operator.
-func Start(controllerMtx *sync.RWMutex) error {
```

Comment:
how about removing code related to mtx?

Reply:
This is a concern for me too. I am thinking about some solution here...

Reply:
As discussed -- this should not be a problem